presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestDistributedQueries.java

```diff
@@ -764,10 +767,25 @@ public void testQueryLoggingCount()
         });
     }
 
+    private <T> T waitUtilStable(Supplier<T> computation, Duration timeout)
+    {
```

Comment:
Since this method is only used for getting the before completed queries count, should we make it more specific like `getBeforeCompletedQueriesCount`? This is currently pretty generic and currently we don't need it to be. 

I also feel like we should add a comment here that summarizes what's available in the git commit message. It's usually easier for the reader to just see it in the code rather than digging into the commit message.

Reply:
I added comment. Not convinced to rename the method. It is fairly simple. And it's use in test is pretty self explanatory without need to look into body.
If something I would return it at all and replaced with sleep. See my comment https://github.com/prestodb/presto/pull/9409#discussion_r153002397